geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ClientServerTransactionDUnitTest.java

```diff
@@ -3782,13 +3790,15 @@ public Object call() throws Exception {
       @Override
       public Object call() throws Exception {
         TXManagerImpl mgr = getGemfireCache().getTxManager();
+        mgr.resume(txid);
```

Comment:
After you resume the transaction, the following code trying to resume can be safely removed.